docker-compose.yml

```diff
@@ -32,6 +32,9 @@
       - MYSQL_DATABASE=app_db
       - MYSQL_USER=app_user
       - MYSQL_PASSWORD=1234
+      - MYSQL_ENGINE=mysql.connector.django
+      - MYSQL_HOST=mysql
```

Comment:
The `MYSQL_HOST` variable is set to `mysql`, which is the default service name resolution in Docker Compose. This variable might be redundant unless used specifically by your application configuration.